gradle/libs.versions.toml

```diff
@@ -39,8 +39,8 @@ jna = "5.13.0"
 core-crypto = "1.0.0-rc.12"
 core-crypto-multiplatform = "0.6.0-rc.3-multiplatform-pre1"
 completeKotlin = "1.1.0"
-desugar-jdk = "2.0.3"
-kermit = "1.2.2"
+desugar-jdk = "1.1.5"
```

Comment:
Why downgrading one lib?

Reply:
IIRC, we can't downgrade to `1.x` on develop without risking some crashes during runtime.

Reply:
Automatic cherry-pick did that, it was supposed to just update `kermit`, I didn't notice that it also changed the one above, sorry about that. Reverted.